src/realm/query_expression.hpp

```diff
@@ -2301,7 +2319,7 @@ class BacklinkCount : public Subexpr2<Int> {
         : m_link_map(std::move(link_map))
     {
     }
-    BacklinkCount(ConstTableRef table, std::vector<ColKey> links = {})
+    BacklinkCount(ConstTableRef table, std::vector<ExtendedColumnKey> links = {})
```

Comment:
nit: why don't `const std::vector<ExtendedColumnKey>& links`?

Reply:
`std::vector<ExtendedColumnKey>&&` is even better